opentelemetry-sdk/src/opentelemetry/sdk/_metrics/view.py

```diff
@@ -0,0 +1,154 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from fnmatch import fnmatch
+from logging import getLogger
+from typing import Optional, Set, Type
+
+from opentelemetry._metrics.instrument import Instrument
+from opentelemetry.sdk._metrics.aggregation import Aggregation
+
+_logger = getLogger(__name__)
+
+
+class View:
+    def __init__(
+        self,
+        instrument_type: Optional[Type[Instrument]] = None,
+        instrument_name: Optional[str] = None,
+        meter_name: Optional[str] = None,
+        meter_version: Optional[str] = None,
+        meter_schema_url: Optional[str] = None,
+        name: Optional[str] = None,
+        description: Optional[str] = None,
+        attribute_keys: Set[str] = None,
+        aggregation: Optional[Type[Aggregation]] = None,
+    ):
+        """
+        An instance of `View` is an object that can perform two actions:
```

Comment:
It doesn't perform these actions though. Maybe "`View`'s configuration parameters are either used as instrument selection criteria or to customize the output metric streams`

In my prototype, I had a separate dataclass for selection criteria to make it more clear in our code/user code

```py
View(
    name="request_count1",
    selector=ViewSelector(instrument_name="app.request_count"),
)
```

imo it's easier to read

Reply:
+1 for `ViewSelector`/`ViewInstrumentSelector` datacalss.

Reply:
I have updated the documentation. I don't think it is convenient to add another class (`ViewSelector`), it makes the API bigger and it is not necessary.